.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules/
 *_pb.js
 lib/
+package-lock.json
```

Comment:
Why did you add this?
I thought this should be added to git:
https://stackoverflow.com/a/44210813/8548472

Reply:
That must of been an accident. I don't remember how it was added.

Reply:
Should we remove it in this PR, or should we merge first and fix that later?

Reply:
I’ll update this PR today then we can merge.

Reply:
Actually, I have something weird going on so I'm going to merge this and I'll fix it with a new PR